### ISSUE TITLE
database update / index not working

### ISSUE THREAD

**Issue by julia-sprenger** [brown170/x4i#35]
Hi @brown170,
I am having issues using the entry index of x4i. When trying to find a an entry using the [get-exfor-entry.py](bin/get-exfor-entry.py) or any of the other utility scripts located in `bin`, I run into an OSError: `Rebuild your index with setup-exfor-db.py, x4i could not find entry O1941` . Running `install-exfor-db.py` and  `setup-exfor-db-index.py` seems to complete successfully, but does not change the error described above.

This error occurs when running locally on my machine as well as in github actions, see https://github.com/julia-sprenger/x4i/actions/runs/10261906031/job/28390529432.

**Comment by julia-sprenger** [comment]
TLDR: The index building is not the issue, tests are passing in a new feature branch.

 I created a [new branch](https://github.com/julia-sprenger/x4i/tree/toml_only_add/integrationCI) including the integration tests as well as the major refactoring I did.

There the index building seems to work, [integration tests]( https://github.com/julia-sprenger/x4i/actions/workflows/integration_test.yml) are passing now.
However for now I switched to using the get-exfor-entry as testscript and didn't check the other scripts yet. But at least the issues doesn't seem to be on the index building side.

The branch only containing the integration CI related changes is [here](https://github.com/julia-sprenger/x4i/tree/add/integrationCI).

**Comment by brown170** [comment]
@julia-sprenger Can I close this issue?
